crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -254,6 +254,19 @@ def prepare(options = {})
 
       protected
 
+      def finalize_cluster_upgrade(cluster)
+        save_upgrade_state("Finalizing the upgrade of nodes in cluster #{cluster}")
+
+        NodeObject.find("pacemaker_config_environment:#{cluster}").each do |node|
+          if node["pacemaker"]["founder"] && node.roles.include?("neutron-network")
+            node.run_ssh_cmd("source /root/.openrc; neutron-ha-tool --l3-agent-rebalance")
```

Comment:
Can we actually have multiple commands in `run_ssh_cmd` (as in: "does it take arbitrary bash code?")